.github/workflows/python.yaml

```diff
@@ -73,5 +81,9 @@ jobs:
         run: poetry run pylint $(git ls-files '*.py')
 
       - name: Run tests
-        if: ${{ inputs.testsMustPass }}
+        if: ${{ inputs.testsMustPass && inputs.codeCoverageFailUnder == 0 }}
```

Comment:
Why is `inputs.codeCoverageFailUnder == 0`?

Reply:
I figured we should add this instead of an `inputs.rankCodeCoverage: bool`. There could be a time when we don't want to add the `pytest-cov` dependency or run/rank coverage at all using `pytest-cov` and I figured this was an easy way to have this be the case. 

For example in a repo where the `coverage` will fail one can simply add `inputs.codeCoverageFailUnder=0`

Reply:
But don't you want a `!=` here

Reply:
No my logic was that setting `== 0` would be the case when we skipped this. `!= 0` is in the lines below where the coverage is ran. So basically if the user of the workflow sets the `rankCodeCoverage: <any_int_other_than_0>` code coverage will be run. However, if they want to skip it completely can set to `0`.

Reply:
Have updated the comments to make this clearer.